### DISCUSSION TITLE
Ask for Data Support

### DISCUSSION THREAD

**Discussion by dinhminhchau** [lkev/wtphm#8] [General]
Dear Dr.  Kevin Leahy

My name is DINH from Changwon National University, South Korea.

I sent you an email to kevin.leahy@umail.ucc.ie, but it seems that you did not use that email address anymore.

I send this message to ask whether you could share the SCADA that you used for your previous work.

For example)  Data used for this paper "A Robust Prescriptive Framework and Performance Metric for Diagnosing and Predicting Wind Turbine Faults Based on SCADA and Alarms Data with Case Study" or Data in the Github Project

I would appreciate it If you could share it.

Thank you

**Comment by lkev** [comment]
Hi Dinh,

Apologies I check that email very rarely. Unfortunately I cannot share any of the datasets I used in my work. I signed NDAs with both companies and no longer have access to the data.

I found it tough to find data but I would suggest just contacting as many wind energy companies as you can find - I had more luck with smaller local operators than large companies and OEMs.

Thanks,
Kevin

**Reply by dinhminhchau** [reply]
I got it. Thank you Kevin.
